### PR TITLE
Dashboard feature

### PR DESCRIPTION
**Pull request by LimbertVargas**

### CODE REVIEW

File: src/main/java/pivotaltracker/ui/Permalink.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
  */
 public final class Permalink {
     public static final String LOGIN_PAGE = "/signin";
-    public static final String DASHBOARD_PAGE = "/dashboard";
+    public static final String DASHBOARD_PAGE = "/workspace";
```
Comment (Cxristian) [inline src/main/java/pivotaltracker/ui/Permalink.java]: why you page workspace called DASHBOARD_PAGE ?

File: src/main/java/pivotaltracker/ui/pages/DashboardPage.java [hunk 2]
```diff
@@ -26,6 +26,15 @@
     @FindBy(css = "[aria-label='Profile Dropdown']")
     private WebElement profileBtn;
 
+    @FindBy(css = "[class='Dashboard__Tabs__tab']")
+    public WebElement projectTab;
```
Comment (Cxristian) [inline src/main/java/pivotaltracker/ui/pages/DashboardPage.java]: Do these web elements need to be public?

File: src/main/java/pivotaltracker/ui/pages/DashboardPage.java [hunk 3]
```diff
@@ -26,6 +26,15 @@
...
+
+    @FindBy(css = "[class='Dashboard__Tabs__tab Dashboard__Tabs__tab--active']")
+    public WebElement workspaceTab;
+
+    @FindBy(css = "[class='content clearfix']")
+    public WebElement dashboardForm;
+
     @Override
     protected void waitUntilPageObjectIsLoaded() {
```
Comment (Cxristian) [inline src/main/java/pivotaltracker/ui/pages/DashboardPage.java]: you need a web elemnt for expected for wait

File: src/main/java/pivotaltracker/ui/pages/workspace/WorkspaceTrackerPage.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+/**
+ * WorkspaceTrackerPage class
+ *
+ * @author Limbert Vargas
+ * @version 1.0
+ */
+public class WorkspaceTrackerPage extends BasePage {
+    @FindBy(id = "tracker")
+    private WebElement workspaceTracker;
```
Comment (Cxristian) [inline src/main/java/pivotaltracker/ui/pages/workspace/WorkspaceTrackerPage.java]: what is workspace traker??

File: src/test/java/steps/DashboardSteps.java [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+ * DashboardSteps class
+ *
+ * @author Limbert Vargas
+ * @version 1.0
+ */
+public class DashboardSteps {
+WorkspaceTab workspaceTab;
+    @When("I Create a (.*) on Project tab with following data")
+    public void iCreateAProjectOnProjectTabWithFollowingData(final String dashboardRef) {
+
```
Comment (Cxristian) [inline src/test/java/steps/DashboardSteps.java:83]: You can't go up empty methods...

File: src/test/java/steps/WorkspaceSteps.java [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+        this.workspace = context.getWorkspace();
+    }
+
+    /**
+     * Creates a new workspace sending the information.
+     *
+     * @param nameWorkspace contains the workspace values.
+     */
+    @When("I create a new workspace (.*)")
+    public void iCreateANewWorkspace(final String nameWorkspace) {
```
Comment (Cxristian) [inline src/test/java/steps/WorkspaceSteps.java]: you name of method is inconsistent with code conventions of java...

File: src/main/java/pivotaltracker/ui/pages/workspace/WorkspaceTrackerPage.java (hunk 4 above)
Comment (LimbertVargas) [reply src/main/java/pivotaltracker/ui/pages/workspace/WorkspaceTrackerPage.java]: is the page that shows when I create a new workspace page

File: src/main/java/pivotaltracker/ui/pages/DashboardPage.java (hunk 3 above)
Comment (LimbertVargas) [reply src/main/java/pivotaltracker/ui/pages/DashboardPage.java]: solve

File: src/main/java/pivotaltracker/ui/pages/DashboardPage.java (hunk 2 above)
Comment (LimbertVargas) [reply src/main/java/pivotaltracker/ui/pages/DashboardPage.java]: solve

File: src/main/java/pivotaltracker/ui/Permalink.java (hunk 1 above)
Comment (LimbertVargas) [reply src/main/java/pivotaltracker/ui/Permalink.java]: solve

File: src/test/java/steps/DashboardSteps.java (hunk 5 above)
Comment (LimbertVargas) [reply src/test/java/steps/DashboardSteps.java:83]: erase

File: src/test/java/steps/WorkspaceSteps.java (hunk 6 above)
Comment (LimbertVargas) [reply src/test/java/steps/WorkspaceSteps.java]: I don't understand your observation

File: .gitignore [hunk 7]
```diff
@@ -35,3 +35,4 @@ out/
 .idea/
 log/
 target/
+logs/
```
Comment (Cxristian) [inline .gitignore:37]: why we need ignored the logs???

File: PivotalTracker.iml [hunk 8]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (Cxristian) [inline PivotalTracker.iml]: do you use this file???

File: src/main/java/pivotaltracker/ui/pages/workspace/WorkspacePopup.java [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+
+    @FindBy(className = "zWDds__Button CMa1J__Button--open")
+    private WebElement cancelWorkspaceBtn;
+
+    /**
+     * Waits until page object is loaded.
+     */
+    @Override
+    public void waitUntilPageObjectIsLoaded() {
+        Log.getInstance().getLog().info("Wait for Crete Workspace Popup upload in the Browser");
```
Comment (Cxristian) [inline src/main/java/pivotaltracker/ui/pages/workspace/WorkspacePopup.java]: do you need save un wait?

File: src/test/resources/features/Dashboard.feature [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+  Then I verify the user name will be shown on the top bar
+
+#  Scenario: Create project
+#    When I go to the Dashboard Page
+#      And I Create a Project on Project tab with following data
+#        | Project Name | Project1             |
+#        | Account      | Test Mail's Projects |
+#    Then I should see the project in Project Page.
+#    When I go to the Dashboard Page
+#    Then I should see the project in Dashboard Page.
```
Comment (Cxristian) [inline src/test/resources/features/Dashboard.feature]: cannot upload commented lines

File: src/test/java/steps/WorkspaceSteps.java (hunk 6 above)
Comment (LimbertVargas) [reply src/test/java/steps/WorkspaceSteps.java]: solve

File: src/test/resources/features/Dashboard.feature [hunk 11]
```diff
@@ -0,0 +1,30 @@
+Feature: Manage Dashboard
+Background:
+  Given I go to the Login Page
+  When I fill the field with credentials from user "Regular User1"
+  Then I verify the user name will be shown on the top bar
+
+#  Scenario: Create project
```
Comment (titor-oopart) [inline src/test/resources/features/Dashboard.feature]: delete this scenario

**Pull request merged**